src/main/java/com/codeborne/selenide/LocalStorageConditions.java

```diff
@@ -0,0 +1,15 @@
+package com.codeborne.selenide;
+
+import java.util.function.Predicate;
+
+public class LocalStorageConditions {
+
+  public static Predicate<LocalStorage> itemPresent(String item) {
```

Comment:
For the sake of readability, it'd better rename it to `containItem()` or `item()` so that the user's code would read like this:

`localStorage().should(containItem("mouse"),"Item 'mouse' doesn't exist")`

`localStorage().shouldHave(item("mouse"),"Item 'mouse' doesn't exist")`

Reply:
you right)
renamed itemPresent -> containItem

Reply:
`containItem()`. 

`containsItem()` reads grammatically incorrect if you consider it as a part of *"should containS item"*.

Reply:
sure?
what about `containsKey` in Map?

Reply:
You right, i followed `containsKey` in `Map` convention, but in case of `should` it looks better)
done